# Move an AGV from an assembly station to a kitting station
# No scoring is applied to the shipment

---
bool success
string message
